examples/7.QMC/main.jl

```diff
@@ -44,37 +48,33 @@ atoms = generate_sites(ChainLattice(), nsites, scale = 5.72)
 
 Ω = 2π * 4
 Δ = 2π * 10
-h = rydberg_h(atoms; Δ, Ω)
+h = rydberg_h(atoms; Δ, Ω);
```

Comment:
why stop the printing here? It should give users a basic idea what Hamiltonian they are working on?

Reply:
Sure